contacts.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Git practice</title>
+    <link rel="stylesheet" href="main.css">
+</head>
+<body>
+    <header>
+        <h1>Science Magazine</h1>
+        <nav>
+            <ul>
+                <li><a href="index.html">HOME</a></li>
+                <li><a href="about.html">ABOUT</a></li>
+                <li><a href="#">REVIEWS</a></li>
+                <li><a href="#">CONTACTS</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <article>
+            <h2>Popular science magazine OYLA</h2>
+            <h3>Contact Us</h3>
+            <h4>Information about the publisher</h4>
+            <address>
+                <p>Popular science magazine OYLA is owned and operated by OYLA MAGAZINE LTD (COMPANY NUMBER: 12718869).<br>128 City Road, London, EC1V 2NX, United Kingdom.<br><br><strong>Tel:</strong> +44 20 4577 04 07<br><strong>Working hours:</strong> 9-5pm, Mon-Fri</p>
+            </address>
+        </article>
+        <hr>
+        <article>
+            <h4>Subscriptions issues</h4>
+            <p>If you have a query about a subscription or back issue (concerning a new order, an existing order or a change of address or other details), please contact<br> our customer services team:<br><strong>Your personal account:</strong><br><br><strong>E-mail:</strong>subscription@oyla.uk<br><strong>Phone:</strong> +44 20 4577 04 07<br><strong>The fastest way to contact us is to message: </strong>Messenger </p>
```

Comment:
1. Indentation issues as mentioned in previous comment need to be improved here as well